src/queries/useAuthorityExport/useAuthorityExport.js

```diff
@@ -0,0 +1,44 @@
+import { useMutation, useQueryClient } from 'react-query';
+import { useOkapiKy, useNamespace } from '@folio/stripes/core';
+
+const useAuthorityExport = ({ onError, onSuccess, ...restOptions }) => {
+  const ky = useOkapiKy();
+  const queryClient = useQueryClient();
+  const [namespace] = useNamespace({ key: 'authorities' });
+
+  const customOptions = {
+    onError,
+    onSuccess: async (response) => {
+      // Creating a delay because result list takes some time to update
+      await new Promise((resolve) => setTimeout(resolve, 500));
+      const data = await response.json();
+
+      queryClient.invalidateQueries(namespace);
```

Comment:
We don't have to invalidate queries here because export doesn't affect authority records.
We do it when editing/deleting because we want to refresh results list after modifying records